packages/react-popover/src/PopoverCustomTrigger.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Popover, PopoverSurface } from './index';
+import type { PopoverProps } from './index';
+import { Button } from './utils.stories';
+import { ExampleContent } from './utils.stories';
+
+export const CustomTrigger = () => {
+  const [open, setOpen] = React.useState(false);
+  const [target, setTarget] = React.useState<HTMLElement | null>(null);
+
+  const onClick = () => setOpen(s => !s);
+  const onOpenChange: PopoverProps['onOpenChange'] = (_, data) => setOpen(data.open || false);
+
+  return (
+    <>
+      <Button aria-haspopup ref={setTarget} onClick={onClick}>
+        Custom trigger
+      </Button>
```

Comment:
There is something wrong with example:
- click on a trigger => popover opens
- click on a trigger => popover stays opened

Can you please check?

Reply:
Yeah need to handle events on the custom trigger separately good catch 👍